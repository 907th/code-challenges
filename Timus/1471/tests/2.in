9
8 0 5
0 3 3
4 6 8
0 4 1
1 7 1
1 5 6
3 1 1
3 2 5
10
2 6
1 6
5 7
8 8
8 6
8 7
3 1
0 2
0 7
2 7
