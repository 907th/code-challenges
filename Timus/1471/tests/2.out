17
13
7
0
14
10
1
8
5
7
